students/gotem888/hw-1/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HW-1</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="wrapper">
+      <button id="plus" class="plus">PLUS</button>
+      <div id="sum" class="sum">0</div>
+      <button id="minus" class="minus">MINUS</button>
+    </div>
+    <script>
+      const plus = document.getElementById("plus");
+      plus.addEventListener("click", (e) => {
+        e.target = n++;
```

Comment:
1) You don't need to use event object for this callback =)
```suggestion
       n++;
```
2) `n` is undeclared for this scenario